packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -293,6 +295,26 @@ private static Point toPoint(Object o, float density) {
     return new Point(toPixels(data.get(0), density), toPixels(data.get(1), density));
   }
 
+  public static CozyMarkerData toCozyMarkerData(Object o){
+    final Map<?, ?> data = toMap(o);
```

Comment:
Is there any way we can define the types of this map here, to make it more intuitive?
For example:
`final Map<String, Object> data = toMap(o);`

Reply:
I just copied it from the plugins itself, but I will be more specific here, thanks!

Reply:
To change this, I would need to change the function as well (otherwise it would be just a cast), and it would change in a lot other places. Seeing the commit log of plugins repo, it has been like that for a while (maybe for some reason), so I will not bother to change this and possibly break something

Reply:
Got it. No problem!